src/utils/fetchClient.ts

```diff
@@ -0,0 +1,41 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+function wait(delay: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
+
+  type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
+
+function request<T>(
+  url: string,
+  method: RequestMethod = 'GET',
+  data: any = null,
```

Comment:
try to fix linter warnings, fix all cases
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/70009205/6f8b38be-32b7-478e-ba8e-ab96d85b69db)